group-membership-form/README.md

```diff
@@ -0,0 +1,103 @@
+---
+title: Offer a one stop shop onboarding experience
+description: Set new hires up for success with the resources they need
+labels:  Sheets, Forms, Groups for Business, Apps Script
+material_icon: settings_input_component
+create_time: 2019-08-30
+update_time: 2019-09-17
+---
+Contributed by Tech and Eco, follow me on
+[Twitter](https://twitter.com/TechandEco)!
+
+It’s hard to give incoming peers a consistently good experience when joining
+the team because of how error prone giving access to different resources can
+sometimes be. By using a Google Group, and sharing resources with that group's
+address, it makes it easy to help new hires get access to what they need to be
+productive in one step.
+
+If you have permissions to add users to a group, you
+can use this script to distribute the responsability to other members of your
+team to fill out the form as you, and have the new person's email added to
+the group, and be granted access to the resources the group is permitted.
+
+For larger teams, you can optionally enable notficiations in the spreadsheet
+to receive an email everytime a new form submission occurs.
+
+## Technology highlights
+
+- The _onFormSubmit_ installable trigger is used to run the script upon
+  a user submitting an entry in a Google Form. To learn more visit
+  [this page](https://developers.google.com/apps-script/reference/script/form-trigger-builder#onFormSubmit())
+- Add members to a _Google Group for Business_ using the
+  [Admin Directory API](https://developers.google.com/apps-script/advanced/admin-sdk-directory)
```

Comment:
is this officially called the Admin SDK?

Reply:
Yea if you look at the website it's called "Admin SDK Directory Service"